assets/syntaxes/Packages/PHP/PHP.sublime-completions

```diff
@@ -284,8 +284,8 @@
 		{ "trigger": "cal_from_jd", "contents": "cal_from_jd(${1:jd}, ${2:calendar})" },
 		{ "trigger": "cal_info", "contents": "cal_info()" },
 		{ "trigger": "cal_to_jd", "contents": "cal_to_jd(${1:calendar}, ${2:month}, ${3:day}, ${4:year})" },
-		{ "trigger": "calcul_hmac", "contents": "calcul_hmac(${1:clent}, ${2:siretcode}, ${3:price}, ${4:reference}, ${5:validity}, ${6:taxation}, ${7:devise}, ${8:language})" },
-		{ "trigger": "calculhmac", "contents": "calculhmac(${1:clent}, ${2:data})" },
+		{ "trigger": "calcul_hmac", "contents": "calcul_hmac(${1:client}, ${2:siretcode}, ${3:price}, ${4:reference}, ${5:validity}, ${6:taxation}, ${7:devise}, ${8:language})" },
```

Comment:
I made a bad guess about what this was... It is or was apparently a thing:

https://web.archive.org/web/20080504115717/https://www.php.net/manual/en/function.calcul-hmac.php

But, I can't really figure out how to find it.